libraries/aws_rds_snapshot_attributes.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsRdsSnapshotAttributes < AwsResourceBase
```

Comment:
I think this can be part of the `AwsRdsSnapshots` as additional attributes rather than being a separate resource

Reply:
I don't think that is possible since 'db_snapshot_identifier' is a mandatory attribute required to get attributes.

Reply:
It takes snapshot id as a mandatory attribute, so I believe this could be treated as a separate resource ?

Reply:
added. Thanks!